config-live-example.toml

```diff
@@ -21,7 +21,7 @@ blocklist_file_path = "./blocklist.json"
 dry_run = true
 dry_run_validation_url = "http://localhost:8545"
 
-blocks_processor_url = "http://block_processor.internal"
```

Comment:
Did you try to run this on the playground? I think the value is not optional and if it is empty it is going to fail because the url is malformed (e.g. empty).

Reply:
Agree, it's Option<> so we just have to remove the line (or comment and add another comment explaining)

Reply:
fixed